controller/telemetry/server.go

```diff
@@ -397,9 +398,17 @@ func responseLabelsFor(responseScope *write.ResponseScope, eosScope *write.EosSc
 	classification := "failure"
 	switch x := eosScope.Ctx.End.(type) {
 	case *write.EosCtx_GrpcStatusCode:
+		// The stream ended with a `grpc-status` trailer.
+		// Classify based on the gRPC status code.
 		if x.GrpcStatusCode == uint32(codes.OK) {
 			classification = "success"
 		}
+	case *write.EosCtx_Other:
+		// The stream did not end with a `grpc-status` trailer (i.e., it was
+		// not a gRPC message). Classify based on the response's HTTP status.
+		if responseScope.Ctx.HttpStatusCode < http.StatusBadRequest {
```

Comment:
This classifies all http 4xx and 5xx responses as failures. My preference is to only classify 5xx errors failures, but that's just, like, my opinion.

Reply:
4xx should not be considered failures by default

Reply:
Okay, will revise.

Reply:
Does it make sense to have a third state?

I know in Firefox Accounts, while 5xxs were super bad, we did have monitoring if 4xxs ever saw a large increase, as it likely meant we changed an API and a consumer wasn't updated to the new usage...

Reply:
that's a good point!

Reply:
@seanmonstar I've made the changes @olix0r and @klingerf requested in 7bde6efe7e11f4843a7ca35865d7b9efb3be71f6. 

It's worth noting that we also label the response with the actual HTTP status code, regardless of what we classify it as:
https://github.com/runconduit/conduit/blob/7bde6efe7e11f4843a7ca35865d7b9efb3be71f6/controller/telemetry/server.go#L413-L416
so eventually, the UI could also surface information about 4xx (and 3xx) codes as well (say, a graph of individual statuses, or grouped by the first digit).